101-acs-swarm/azuredeploy.parameters.json

```diff
@@ -18,7 +18,7 @@
       "value": "SwarmPreview"
     },
     "masterCount": {
-      "value": "3"
+      "value": 3
```

Comment:
The CI output error is for another error (in metadata as per previous line comment).